torch/_inductor/ir.py

```diff
@@ -5529,7 +5561,40 @@ def __init__(
             accum_zp, o_inv_scale, o_zp, fp32_output, binary_attr, aplha, unary_attr, unary_scalars, unary_algorithm]
         """
         self.has_bias = len(inputs) == 6
-        super().__init__(layout, inputs, constant_args)
+        super().__init__(
+            layout,
+            inputs,
+            constant_args,
+            None,
+            kernel="torch.ops.onednn.qconv2d_pointwise.binary",
+            cpp_kernel="onednn::qconv2d_pointwise",
+        )
+        self.cpp_kernel_overlad_name = "binary"
+        self.cpp_kernel_key = "qconv2d_pointwise_binary"
+        self.cpp_op_schema = """
```

Comment:
Is it possible to read this kind of schema from somewhere instead of keep adding them for more and more ops?

Reply:
Let me submit a subsequent PR to clean this up. Created an issue to track it: https://github.com/pytorch/pytorch/issues/112552